### PR TITLE
Create Jupyter Notebooks post

### PR DESCRIPTION
**Pull request by atribolet**
Ready to be published after you make any additional needed changes. Thanks!

### CODE REVIEW

File: _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+author:
+  display_name: Keaton Walker
+  email: kwalker@utah.gov
+date: 2019-06-25 21:29:10
+categories:
+  - Featured
+tags:
+  - notebook
+  - jupyter
+  - pyton
```
Comment (steveoh) [inline _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md]:
```suggestion
  - python
```

File: _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: 'Using Jupyter Notebook for AGRC Collaboration'
+author:
+  display_name: Keaton Walker
+  email: kwalker@utah.gov
+date: 2019-06-25 21:29:10
+categories:
+  - Featured
```
Comment (steveoh) [inline _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md]:
```suggestion
  - Featured
  - Developer
```

File: _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Working with log data, or any other source of data, can involve multiple data transformations. Sometimes many transformations and analysis need to be applied to data, and the work is distributed across multiple people in our office. When more than one person is involved in a process, good communication and documentation become much more important. This [notebook created with the ArcGIS Pro environment](https://github.com/keatonwalker/multimodal/blob/master/gtfs_parsing/trip_stops.ipynb) shows work done for our current [multi-modal project](https://gis.utah.gov/multi-modal-travel-shed-analysis-project/). For this project, the general transit feed specification (GTFS) files produced by the Utah Transit Authority need to be transformed into a format that would work with ESRI’s Network Analyst while maintaining the ID structure of the original GTFS. The notebook shows how we parsed the GTFS files and transformed them into a shapefile format that could be consumed by ArcGIS. The notebook was created with the ArcGIS Pro installation of Python and was then pushed to Github. Github has a very nice feature of rendering notebooks with outputs and code shown in the Github web application.
```
Comment (steveoh) [inline _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md]: links should never contain `gis.utah.gov`. An example https://github.com/agrc/gis.utah.gov#link-to-post

```suggestion
Working with log data, or any other source of data, can involve multiple data transformations. Sometimes many transformations and analysis need to be applied to data, and the work is distributed across multiple people in our office. When more than one person is involved in a process, good communication and documentation become much more important. This [notebook created with the ArcGIS Pro environment](https://github.com/keatonwalker/multimodal/blob/master/gtfs_parsing/trip_stops.ipynb) shows work done for our current [multi-modal project]({{site.baseurl}}{% post_url 2018-11-07-multi-modal-travel-shed-analysis-project %}). For this project, the general transit feed specification (GTFS) files produced by the Utah Transit Authority need to be transformed into a format that would work with ESRI’s Network Analyst while maintaining the ID structure of the original GTFS. The notebook shows how we parsed the GTFS files and transformed them into a shapefile format that could be consumed by ArcGIS. The notebook was created with the ArcGIS Pro installation of Python and was then pushed to Github. Github has a very nice feature of rendering notebooks with outputs and code shown in the Github web application.
```

Comment (atribolet) [reply _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md]: Sorry! Didn't really look too closely at the links Keaton included. Will make note to check them for internal posts next time!

Comment (steveoh) [reply _posts/2019-06-25-using-jupyter-notebook-for-agrc-collaboration.md]: not a problem. i figured that was the case.

**Pull request merged**